Code/CocoaLumberjack/RKLumberjackLogger.m

```diff
@@ -24,7 +24,7 @@ + (DDLogLevel)ddLogLevelFromRKLogLevel:(_RKlcl_level_t)rkLevel
         case RKLogLevelDebug:    return DDLogLevelDebug;
         case RKLogLevelTrace:    return DDLogLevelVerbose;
     }
-
+    
```

Comment:
I have just used `git revert ...` and these must have been cleaned up by one of the original commits
I'll make a new commit to clean up